### ISSUE TITLE
`br_table` validation has outdated bullet point?

### ISSUE THREAD

**Issue by fitzgen** [WebAssembly/spec#1376]
This is with regards to the English prose bullet points, not the typing rules.

It seems like the second bullet point defines `[t*]` concretely but then the fourth bullet point redefines `[t*]` existentially.

https://webassembly.github.io/spec/core/valid/instructions.html#valid-br-table

> * Let `[t*]` be the result type `C.labels[l_N]`.
>
> ...
>
> * There must be a result type `[t*]`, such that: ...

If I understand correctly, the fourth bullet point is correct given the newer subtyping/matching stuff, and the second bullet point is outdated and should be removed.

Am I understanding this correctly?

**Comment by rossberg** [comment]
Yes, you are right, this is a leftover from the MVP version. Fixed in #1378. Thanks for spotting!
